src/physics_swe.c

```diff
@@ -3,6 +3,8 @@
 #include <private/rdymemoryimpl.h>
 #include <stddef.h>  // for offsetof
 
+#include "swe-flux.h"
```

Comment:
Picking a nit: would you mind if we called this `swe_flux.h`? We have some files already that use underscores, and as a victim of the trauma created by C++'s "frivolous decision creation syndrome", I'm sensitive to having multiple ways of naming and referring to things. :-)

Reply:
Sure, are you still hacking on the branch or should I just push?

Reply:
Go ahead and push. Thanks!

Reply:
Okay, now I'm definitely finished!